advocate_europe/settings/base.py

```diff
@@ -251,6 +251,7 @@
     '': {
         'heroimage': {'size': (1500, 500), 'crop': 'smart'},
         'ideatileimage': {'size': (400, 200), 'crop': 'smart'},
+        'ideadetailimage': {'size': (1500, 500), 'crop': 'smart'},
```

Comment:
Or should I use the heroimage tag right away? Thought to make another extra one was better if we want to change sizes later on?

Reply:
Well I think both solutions are fine, lets go with this one.